nats.go

```diff
@@ -146,6 +146,8 @@ var (
 	ErrInvalidJSAck                 = errors.New("nats: invalid jetstream publish response")
 	ErrMultiStreamUnsupported       = errors.New("nats: multiple streams are not supported")
 	ErrStreamNameRequired           = errors.New("nats: stream name is required")
+	ErrNoStream                     = errors.New("nats: no stream found")
```

Comment:
Also maybe change to:
```suggestion
	ErrNoStream                     = errors.New("nats: stream not found")
```

So that it is similar to the error that was returned from server: https://github.com/nats-io/nats-server/blob/77d54dabeb97c4f7e2d1cbe98e4cd33108e71560/server/errors.json#L351-L354